tools/ete/ete_lineage_generator.xml

```diff
@@ -68,17 +69,16 @@ python '$__tool_directory__/ete_lineage_generator.py'
         </conditional>
         <conditional name="compresscond">
             <param name="compress" type="select" label="Fill unnamed ranks" help="Fill unnamed ranks with super/sub ranks (see -l)">
-                <option value="-c">Yes</option>
+                <option value="-c" selected="True">Yes</option>
                 <option value="">No</option>
             </param>
             <when value="-c">
-                <param name="lower" type="select" label="Prefer lower ranks for filling" help="Prefer lower levels for filling">
-                    <option value="-l">Yes</option>
-                    <option value="">No</option>
-                </param>
+                <param name="lower" type="boolean" truevalue="-l" falsevalue="" checked="False" label="Prefer lower ranks" help="Take the next available lower rank (default: higher)"/> 
             </when>
             <when value="" />
         </conditional>
+        <param argument="--includeid" type="boolean" truevalue="--includeid" falsevalue="" label="Include taxid in the table"
```

Comment:
I'd use `name="includeid"` instead of `argument` since this is not an option of the underlying `ete` tool.

Reply:
it is an option of the python script that is called. but if you want I can add an extra `name="includeid"`

Reply:
Yep, but `argument` is supposed to help people cross-reference the Galaxy tool parameters with the options/arguments of the underlying tool, which in this case is `ete`, not our Python script which is part of the wrapping.

Reply:
got it. fixed